app/Services/ScBookingService.cs

```diff
@@ -94,11 +94,11 @@ public ScCaseSearchViewModel LoadSearchForm2()
                 Results = bookingInfo.Results,
                 BookingLocationName = bookingInfo.BookingLocationName,
                 BookingRegistryId = bookingInfo.BookingRegistryId,
-                AvailableConferenceTypeIds = bookingInfo.AvailableConferenceTypeIds,
+                AvailableHearingTypeIds = bookingInfo.AvailableHearingTypeIds,
```

Comment:
You also need to copy the new fields from the session to the viewmodel here.  That's why some values are being lost when you reload step 2.

- EstimatedTrialLength
- IsHomeRegistry
- IsDifferentPlaceOfTrial
- TrialLocation

Reply:
Ah, I added them to this block and it made a difference, although now the values persist even if I go back to the start and pick a different case file. But I think that's just because the navigation goes back to the `LoadSearchForm2` view and not the `LoadSearchForm` which clears the session.